velox/dwio/dwrf/reader/DwrfReader.cpp

```diff
@@ -111,7 +115,11 @@ uint64_t DwrfRowReader::next(uint64_t size, VectorPtr& result) {
       // reading of the data.
       setStrideIndex(strideSize > 0 ? currentRowInStripe / strideSize : 0);
 
-      columnReader_->next(rowsToRead, result);
+      if (selectiveColumnReader_ != nullptr) {
```

Comment:
nit: `if (selectiveColumnReader_ )` should do?

Reply:
@majetideepak Thanks for reviewing! Yes it should work without the inequality check, I was just thinking it'll be more straightforward to read with the check. I removed all the ` != nullptr` anyways.